docs/book/src/topics/crd-changes/v1alpha6-to-v1alpha7.md

```diff
@@ -175,6 +176,31 @@ securityGroupFilters:
 - id: 4a131d3e-9939-4a6b-adea-788a2e89fcd8
 ```
 
+### Change to ports.bindingProfile
```

Comment:
`ports.profile`. This should also be `####`. Wouldn't have noticed if the indentation on the doctoc wasn't off, so thanks for updating that!

```suggestion
#### Change to ports.profile
```